build-tools/libzip/libzip.targets

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_SourceTopDir>..\..</_SourceTopDir>
+  </PropertyGroup>
+  <UsingTask AssemblyFile="$(_SourceTopDir)\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.GetNugetPackageBasePath" />
+  <Import Project="libzip.props" />
+  <Import Project="libzip.projitems" />
+  <Target Name="_Make"
+        Inputs=""
+        Outputs="$(IntermediateOutputPath)\%(_LibZipTarget.Identity)\%(_LibZipTarget.OutputLibraryPath)">
+    <Exec
+        Command="make"
+        WorkingDirectory="$(IntermediateOutputPath)\%(_LibZipTarget.Identity)"
+     /> 
+    <Copy SourceFiles="@(_LibZipTarget->'$(IntermediateOutputPath)\%(Identity)\%(_LibZipTarget.OutputLibraryPath)')"
+        DestinationFiles="@(_LibZipTarget->'$(OutputPath)\lib\xbuild\Xamarin\Android\%(_LibZipTarget.OutputLibrary)')"/>
+  </Target>
+  <Target Name="_Configure">
```

Comment:
Please order `_Configure` before `_Make`, particularly since it's executed first.